crates/swc_css_minifier/src/compressor/rules.rs

```diff
@@ -58,10 +58,25 @@ impl Compressor {
         }
     }
 
-    fn get_declaration_name(&self, declaration: &Declaration) -> JsWord {
-        match &declaration.name {
-            DeclarationName::Ident(Ident { value, .. }) => value.to_ascii_lowercase(),
-            DeclarationName::DashedIdent(DashedIdent { value, .. }) => value.to_ascii_lowercase(),
+    fn is_same_declaration_name(&self, left: &Declaration, right: &Declaration) -> bool {
+        match (&left.name, &right.name) {
+            (
+                DeclarationName::Ident(Ident {
+                    value: left_value, ..
+                }),
+                DeclarationName::Ident(Ident {
+                    value: right_value, ..
+                }),
+            ) => left_value.to_ascii_lowercase() == right_value.to_ascii_lowercase(),
```

Comment:
Let's use `eq_ignore_ascii_case`

Reply:
@kdy1 I will fix, anyway, declaration name will be normalize too in future, just do it step by step :smile: